packages/manager/src/utilities/formatRegion.ts

```diff
@@ -19,5 +26,65 @@ export const getRegionCountryGroup = (region: Region | undefined) => {
     : 'Other';
 };
 
-export const isEURegion = (region: null | string | undefined) =>
-  region?.match('^eu');
+export const getSelectedRegion = (
+  regions: Region[],
+  selectedRegionId: string
+): Region | undefined => {
+  return regions.find((thisRegion) => selectedRegionId === thisRegion.id);
+};
+
+export const getSelectedRegionGroup = (
+  regions: Region[],
+  selectedRegionId: string
+): string | undefined => {
+  const selectedRegion = getSelectedRegion(regions, selectedRegionId);
+
+  if (!selectedRegion) {
+    return undefined;
+  }
+
+  return getRegionCountryGroup(selectedRegion);
+};
+
+export const isEURegion = (regionContinent: string | undefined): boolean => {
+  return regionContinent === CONTINENT_CODE_TO_CONTINENT.EU;
+};
+
+/**
+ *
+ * @param opts.agreements The user's agreements
+ * @param opts.profile The user's profile
+ * @param opts.regions The list of regions
+ * @param opts.selectedRegionId The ID of the selected region (e.g. 'eu-west')
+ * @returns selectedRegionGroup: string | undefined - the group of the selected region, if it exists
+ * @returns showGDPRCheckbox: boolean - whether or not to show the GDPR checkbox
+ * @usage
```

Comment:
`@example` will allow VSCode to add syntax highlighting

```suggestion
 * @example
```